src/MoBi.Assets/ToolTips.cs

```diff
@@ -98,6 +98,8 @@ public static class SimulationRibbon
          public static readonly string RunWithSettingsDescription = "Select simulation settings and simulate the active simulation";
          public static readonly string StopSimulation = "Stop the currently running Simulation";
          public static readonly string CalculateScaleFactors = "Calculate scale factors";
+         public static readonly string ConfigureSimulationDescription = "Configure Simulation";
```

Comment:
In `RunWithSettingsDescription `it's _**s**imulation_,
in `StopSimulation `and `ConfigureSimulationDescription ``it's _**S**imulation_
Should be consistent

Reply:
actually nothing was consistent here. I have renamed everything to be consistent (Simulation, Building Block etc.. all low caps)